tests/header/header_test.cpp

```diff
@@ -45,6 +45,7 @@ class TEST_NAME : public util::test_base {
 #define TEST_FAIL
     log.note("SYCL_LANGUAGE_VERSION not present");
 #else
+    static_assert(SYCL_LANGUAGE_VERSION, "202012L");
     log.note("SYCL_LANGUAGE_VERSION = %d", static_cast<int>(SYCL_LANGUAGE_VERSION));
```

Comment:
Not sure if this log message is still useful since we are now checking for a specific value.

Reply:
As for me, we can remove this log message due to static_assert.